src/components/overall-statistics/tabs/WinratesTab.vue

```diff
@@ -172,7 +183,7 @@ export default class WinratesTab extends Vue {
     if (!statsPerMapAndRace) {
       return [];
     }
-    return statsPerMapAndRace.ratio.slice(1, 5);
+    return statsPerMapAndRace.ratio.slice(1, 5).concat(statsPerMapAndRace.ratio[0]);
```

Comment:
Could you rename the function from raceWinrateWithoutRandom to raceWinrate since it now includes random?